### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AtillaTahak**
Selected the API I am going to use.
Built files structure for My React app.
Prepared routes and navigation in my app.
Created the tests for the application.
Styled my components to match the design provided.
Deployed the project and test for final details.
Recorded a video for my project.
Created a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
+# Loom Video
+[Click here](https://www.loom.com/share/7989bc3a0ddb4740b5f0f01e8c072c7a)
 
-![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+# Screenshot
 
-## Built With
+![Image of the desktop view](img/screenshot.png)
```
Comment (DeliceLydia) [inline README.md]: - [ ] FIx this screenshot please

File: README.md [hunk 2]
```diff
@@ -1,76 +1,76 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Space Travellers
 
-# Project Name
+> This is a React and Redux based single page application about EartQuake.
```
Comment (DeliceLydia) [inline README.md]: - [x] Try to explain what the project is all about in the description and make sure you use the correct grammar like example its `EarthQuake` not `EartQuake`

File: src/components/Home.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          <span>Start Date</span>
+          <input id="end-date" type="date" value={content.startInput} onChange={(e) => content.setStartInput(e.target.value)} />
+        </Elements>
+        <Elements>
+          <span>End Date</span>
+          <input id="end-date" type="date" value={content.endInput} onChange={(e) => content.setEndInput(e.target.value)} />
+
+        </Elements>
+        <Elements>
+          <span>Earth Quake Minumum Magnitute</span>
```
Comment (DeliceLydia) [inline src/components/Home.js]: - [ ] Make sure to use correct word like `Minimum Magnitude`

File: src/components/styles/GlobalStyle.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  background-color: #dddddd;
+}
+
+.home-go-back {
+  position: fixed;
+  top: 25px;
+  right: 50px;
+  font-size: 40px;
+  z-index: 25;
+}
```
Comment (DeliceLydia) [inline src/components/styles/GlobalStyle.js]: - [x] Referring to the requirements you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
- Typography: font face, size, and weight
- Layout: composition and space between elements.

File: README.md (hunk 2 above)
Comment (DeliceLydia) [reply README.md]: - [x] I suggest that you add instructions in the README on how to use your application so that even anyone can understand how to use it

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @AtillaTahak, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! here are highlights for your project:
- I like the idea of your application about earthquake
- No linter errors



## Required Changes ♻️
_Check the comments under the review._



### Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Home.js (hunk 3 above)
Comment (AtillaTahak) [reply src/components/Home.js]: thank for review

File: README.md (hunk 1 above)
Comment (AtillaTahak) [reply README.md]: thank you for review

File: README.md (hunk 2 above)
Comment (AtillaTahak) [reply README.md]: Thank you for review

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**